habitat_baselines/common/construct_vector_env.py

```diff
@@ -0,0 +1,99 @@
+import os
```

Comment:
Moved this file from habitat-lab to habitat-baselines since the API of the VectorEnv is tightly coupled to the trainers. More files need to be moved, but it will be done in subsequent PRs to avoid making this PR too large.